### DISCUSSION TITLE
Export Feeds for Assets

### DISCUSSION THREAD

**Discussion by paultvis** [atrocore/atrocore#565] [Q&A]
Hi,

Am I missing something with export feeds for assets? I have been through the documentation several times now on export feeds and can see references to additional export feed modules (which are not installed but do not seem to appear on the marketplace) and references to additional export types, but these are not available in the Export Feed configuration.

What do we need to do to enable downloading a zip file containing assets?

Thanks

Paul

**Comment by atrolex** [comment]
Currently Assets can be imported/exported only as URLs. We plan to implement the possibility to export archives soon. I think in January. This will be free functionality. No module will be needed.
